packages/nimble/inst/tests/test-mcmc.R

```diff
@@ -2,25 +2,26 @@ source(system.file(file.path('tests', 'test_utils.R'), package = 'nimble'))
 
 context("Testing of default MCMC")
 
-### TODO: add in the special cases for dipper
-
-if(FALSE) { # template for running JAGS for comparison
-  require(R2jags)
-    dir = system.file(file.path('classic-bugs', 'vol2', 'air'), package = 'nimble')
-  data = new.env(); inits = new.env()
-  source(file.path(dir, 'air-data.R'), data)
-  source(file.path(dir, 'air-init.R'), inits)
-  data = as.list(data)
-  inits = list(as.list(inits))
-  out1 <- jags(data = data, inits = inits,
-               parameters.to.save = c('X','theta'), n.chains = 1,
-               n.iter = 100000, n.burnin = 50000, n.thin = 1, model.file = file.path(dir, 'air.bug'),
-               DIC = FALSE, jags.seed = 0)
-  out <- as.mcmc(out1)
-}
-
-
-### Beginning of actual tests
+RwarnLevel <- options('warn')$warn
+options(warn = -1)
+nimbleVerboseSetting <- nimbleOptions('verbose')
+
+## If you do *not* want to write to results files
```

Comment:
👍  This seems to me to be a cleaner way to verify results over time, as compared to our current practice of manually snapshotting test results files for each Cran release.